flixel/addons/plugin/screengrab/FlxScreenGrab.hx

```diff
@@ -155,28 +152,54 @@ class FlxScreenGrab extends FlxBasic
 		{
 			Filename = Filename + ".png";
 		}
+		return Filename;
+	}
+	
+	private static function save(Filename:String = ""):Void
+	{
+		if (screenshot.bitmapData == null)
+		{
+			return;
+		}
+		
+		Filename = fixFilename(Filename);
+		
+		var png:ByteArray;
+	#if flash
+		png = PNGEncoder.encode(screenshot.bitmapData);
+	#elseif openfl_legacy
```

Comment:
There is a difference. IIRC, Joshua told me that if you're trying to test against legacy/next, then you should use "openfl_legacy" if the thing you're guarding against is in an openfl. or flash. package, and "lime_legacy" if it's in an lime. package.